src/components/splash-screen/styles.ts

```diff
@@ -4,7 +4,7 @@ export const styles = {
   container: {
     position: "fixed",
     inset: 0,
-    backgroundColor: "common.white",
+    backgroundColor: "common.black",
     zIndex: 1,
```

Comment:
Maybe we can think about making the z-index a constant and use that throughout the application.

Reply:
hmm, Well that's a little tricky. z indices are dependant on it's relative parent, meaning if a parent's zindex is -1 and the child is 1, it will not overlap the parent. However I did not add this zindex so I will to check later what's used for.